GVFS/GVFS.Common/ProductUpgrader.cs

```diff
@@ -245,13 +245,14 @@ public bool TryCleanup(out string error)
         public virtual bool TryLoadRingConfig(out string error)
         {
             string gitPath = GVFSPlatform.Instance.GitInstallation.GetInstalledGitBinPath();
-            GitProcess.Result result = GitProcess.GetFromGlobalConfig(gitPath, GVFSConstants.GitConfig.UpgradeRing);
-            if (!result.HasErrors && !string.IsNullOrEmpty(result.Output))
+            LocalGVFSConfig localConfig = new LocalGVFSConfig(gitPath);
+
+            string ringConfig = null;
+            if (localConfig.TryGetValueForKey(GVFSConstants.GitConfig.UpgradeRing, out ringConfig, out error))

```

Comment:
Move the `UpgradeRing` constant out of the `GitConfig` class.

Reply:
Moved it to LocalGVFSConfig class